### ISSUE TITLE
Class definition and import errors have unhelpful tracebacks

### ISSUE THREAD

**Issue by jonathanverner** [brython-dev/brython#649]
When I erroneously define a class as follows:

```python
class Test:
    T = 1
    def __init__(self, a=Test.T):
        pass
```

Brython gives me the following:

```python
Internal Javascript error: TypeError: <TypeError: Cannot read property '__class__' of undefined>
```

whereas CPython gives:

```python
Traceback (most recent call last):
  File "<pyshell#44>", line 1, in <module>
    class Test:
  File "<pyshell#44>", line 3, in Test
    def __init__(self, a=Test.T):
NameError: name 'Test' is not defined
```
which is much more helpful.

Also, importing a module which contains this class gives 

```python
>>> import mod
Traceback (most recent call last):
  module console line 194
    traceback.print_exc(self)
  module <module> line 1
    import mod TestImportError: cannot import name 'mod'
>>> 
```

The __same__ message is given even when the offending class is not in the module `mod` itself, but is (recursively) imported by the module `mod`. This makes tracking down the error much harder as one needs to go through all the (recursively) imported modules to find out which is failing and then one needs to step through the generated javascript to find out, where the error occurs.

**Comment by PierreQuentel** [comment]
The commit referenced above fixes the first part of the issue : the error message is now the same as with CPython. The second part remains open.

**Comment by PierreQuentel** [comment]
I think the second part of this issue (more explicit message if an import fails because of an error in an imported module) is fixed by commit 7c505a764ed923d4cf7316d601cc5cf2c0b8bfe3. I close the bug, but feel free to reopen it if necessary
